### PR TITLE
Fix SSSS overwrite if user never enters their SSSS key

### PR DESCRIPTION
**Pull request by foldleft**
fixes: https://github.com/vector-im/riot-web/issues/12352

This bug caused the cross-signing panel to indicate that there were no cross-signing keys available if the user did not provide any keys, even though cross-signing is set up.  This allowed the user to re-initialize the cross-signing store, destroying whatever was there previously.

**Before giving keys to the client**
Screenshot 2020-02-21 at 10 12 57

**After giving keys to the client**
Screenshot 2020-02-21 at 10 13 21

### CODE REVIEW

**Comment by turt2live** [comment]
@foldleft can this PR get a title that makes sense in a changelog please?

File: src/components/views/settings/CrossSigningPanel.js [hunk 1]
```diff
@@ -128,7 +128,6 @@ export default class CrossSigningPanel extends React.PureComponent {
         }
 
         const enabled = (
-            crossSigningPublicKeysOnDevice &&
```
Comment (dbkr) [inline src/components/views/settings/CrossSigningPanel.js]: Isn't this going to cause the `summarisedStatus` status to say it's enabled when it isn't?

Comment (foldleft) [reply src/components/views/settings/CrossSigningPanel.js]: weirdly, `crossSigningPublicKeysOnDevice` seems to only be true if the _private_ keys are also on the device.  Even if the device doesn't have keys, the cross-signing can be initialised.

**Comment by jryans** [comment]
@foldleft Please also use title case on PRs for the changelog.

File: src/components/views/settings/CrossSigningPanel.js (hunk 1 above)
Comment (dbkr) [reply src/components/views/settings/CrossSigningPanel.js]: Yeah, `crossSigningPublicKeysOnDevice` is whether the public key is trusted or not. My point was that in the if block just below, we'll now only hit the second case if `crossSigningPrivateKeysInStorage` is true but `secretStorageKeyInAccount` is false. So this corrects the second if block but breaks the first?

**Pull request closed without merging**